packages/nc-gui/helpers/tiptap/extensions/tiptap/markdown.ts

```
@@ -0,0 +1,70 @@
+      parser: new MarkdownParser(this.editor, this.options),
+      serializer: new MarkdownSerializer(this.editor),
+      getMarkdown: () => {
+        return this.editor.storage.markdown.serializer.serialize(this.editor.state.doc)
+      },
+    }
+    this.editor.options.initialContent = this.editor.options.content
+    this.editor.options.content = this.editor.storage.markdown.parser.parse(this.editor.options.content)
+  },
+  onCreate() {
+    this.editor.options.content = this.editor.options.initialContent
+    delete this.editor.options.initialContent
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)